bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolder.java

```diff
@@ -189,6 +189,7 @@ public class CTabFolder extends Composite {
 	boolean selectionGradientVertical;
 	Color selectionForeground;
 	Color selectionBackground;
+	int selectionHighlightBarThickness = 0;
```

Comment:
- Suggest we keep default value as 1(or something which enables this feature by default)
- Default value can be revised based on feedback

Reply:
Made it 2 as 1 pixel is to thin to easily distinguish the different colors.

Reply:
I'm still not sure about pushing the changes to the widget by default for the same reasons (as discussed before). But, may be we can keep it on for now to get some testing and feedback.